packages/network/nodesList/manager/manager.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
That file should be a repeater and not a manager.

Reply:
The Table is performing the task of a repeater. I don't have distinct repeater myself.

Reply:
Dear Elnaz,

we try to separate API call logic from the component so the composer is moved to another file called manager so in future we can change the way of API call or reuse the manager